client/my-sites/domains/domain-management/settings/index.tsx

```diff
@@ -56,6 +61,53 @@ const Settings = ( props: SettingsPageProps ): JSX.Element => {
 		</>
 	);
 
+	const getContactInformationAccordion = () => {
```

Comment:
I think there might be a problem here when viewing domains of which we are not owners - the `requestWhois` query will return an error, and the component's loading state will never be resolved. A quick and dirty fix would be adding a condition at the start of this method to check if the current user can manage this domain, something like

```
if ( ! domain.currentUserCanManage ) {
	return (
		<Accordion title="Contact information">
			<ContactsPrivacyInfo
				domains={ props.domains! }
				selectedSite={ props.selectedSite }
				selectedDomainName={ props.selectedDomainName }
			></ContactsPrivacyInfo>
		</Accordion>
	);
}
```

This duplicates some code, but I couldn't think of anything more elegant at the moment 😅